src/cargo/lib.rs

```diff
@@ -1,34 +1,11 @@
 #![cfg_attr(test, deny(warnings))]
 // While we're getting used to 2018:
```

Comment:
Are we still getting use to 2018? If the lines are still useful, lets update the comment.

Reply:
Yea. I updated the comment, though I'm not sure what would be good to say. WDYT?

For reference, the current idiom lints are:

Lint | Default Level
-----|---------------
[bare-trait-objects](https://doc.rust-lang.org/1.51.0/rustc/lints/listing/warn-by-default.html#bare-trait-objects) | warn
[ellipsis-inclusive-range-patterns](https://doc.rust-lang.org/1.51.0/rustc/lints/listing/warn-by-default.html#ellipsis-inclusive-range-patterns) | warn
[unused-extern-crates](https://doc.rust-lang.org/1.51.0/rustc/lints/listing/allowed-by-default.html#unused-extern-crates) | allow
[elided-lifetimes-in-paths](https://doc.rust-lang.org/1.51.0/rustc/lints/listing/allowed-by-default.html#elided-lifetimes-in-paths) | allow
[explicit-outlives-requirements](https://doc.rust-lang.org/1.51.0/rustc/lints/listing/allowed-by-default.html#explicit-outlives-requirements) | allow

The links give explanations, but most of them are allowed due to false positives (which aren't a problem for us).

Also, some of these are changing in 2021.